packages/xeus/src/worker.ts

```diff
@@ -208,7 +208,9 @@ export abstract class XeusRemoteKernel {
 
       this._initializeStdin(baseUrl, browsingContextId);
 
```

Comment:
Maybe we should add a comment, since this `.length` check is not super clear.

```suggestion
      // backward compatibility: Checking if the kernel constructor takes argument or not
```

Reply:
Yupp, I think something like this makes sense.